packages/manager/modules/support/src/tickets/new-ticket/issues-form/issues-form.controller.js

```diff
@@ -111,9 +146,32 @@ export default class SupportNewIssuesFormController {
       external: false,
     }).$promise.then((items) => {
       this.services = items;
+
+      if (this.defaultServiceType != null) {
+        const inputService = this.services
+          .find(service => service.serviceName === this.defaultServiceName);
+
+        if (inputService == null) {
```

Comment:
not sure it matters here but conventionally with use strict comparisons : ===

Reply:
`x == null` is the same as `x === null && x === undefined` -> this is a useful way to make sure a variable is defined :)

Reply:
i commonly use ```if (!inputService) { ... }``` but i guess either way is fine

Reply:
I don't do it because `!value` matches `false`, `''`, '0', and so on